GeosCore/calc_met_mod.F90

```diff
@@ -293,24 +293,23 @@ SUBROUTINE AIRQNT( Input_Opt, State_Chm, State_Grid, State_Met, &
        IsLocNoon(I,J) = ( LocTimeSec(I,J)          <= 43200  .and. &
                           LocTimeSec(I,J) + Dt_Sec >= 43200 )
 
+       ! Snow
+       FRSNOW = State_Met%FRSNOW(I,J)
+
        ! Land without snow or ice
-       FRLAND_NOSNO_NOICE = State_Met%FRLAND(I,J) -                &
-               (State_Met%FRLAND(I,J)*State_Met%FRSNO(I,J))
+       FRLAND_NOSNOW_NOICE = State_Met%FRLAND(I,J) - State_Met%FRSNOW(I,J)
```

Comment:
If this is land without snow or ice, why are we only subtracting FRSNOW? Should we also subtract FRLANDICE?

Reply:
Good point. I'll look into the history of this and see what it needs to be based on usage.

Reply:
I figured out the history for this. `FRLAND_NOSNOW_NOICE` used to subtract `FRLANDIC` but a while back I discovered that was a bug and fixed it in 14.1.0 (https://github.com/geoschem/geos-chem/pull/1425). `FRLAND` in the constants file does not include ice over land, hence there is a separate entry `FRLANDIC` in the constants file. `FRLANDIC` is specifically glaciers since the source is constant. `FRSNOW` is time-varying and so we subtract it from `FRLAND`. I will update the code to be more clear about this.

Reply:
@msulprizio @lizziel: I think this goes back to about GEOS-5 when the LWI fields were superseded by the FR* fields.  Also it should be FRLANDICE but at the time we could only use 8 chars for the container name in the bpch file, so we lopped off the ending "E".